src/server/exemption/site-details/before-you-start/controller.test.js

```diff
@@ -0,0 +1,110 @@
+import { createServer } from '~/src/server/index.js'
+import {
+  beforeYouStartController,
+  BEFORE_YOU_START_SITE_DETAILS_VIEW_ROUTE
+} from '~/src/server/exemption/site-details/before-you-start/controller.js'
+import { getExemptionCache } from '~/src/server/common/helpers/session-cache/utils.js'
+import { mockExemption } from '~/src/server/test-helpers/mocks.js'
+import { statusCodes } from '~/src/server/common/constants/status-codes.js'
+import { config } from '~/src/config/config.js'
+import { JSDOM } from 'jsdom'
+import { routes } from '~/src/server/common/constants/routes.js'
+
+jest.mock('~/src/server/common/helpers/session-cache/utils.js')
+
+describe('#beforeYouStart', () => {
+  /** @type {Server} */
+  let server
+
+  beforeAll(async () => {
+    server = await createServer()
+    await server.initialize()
+  })
+
+  beforeEach(() => {
+    jest.resetAllMocks()
+    jest.mocked(getExemptionCache).mockReturnValue(mockExemption)
+  })
+
+  afterAll(async () => {
+    await server.stop({ timeout: 0 })
+  })
+
+  describe('#beforeYouStartController', () => {
+    test('beforeYouStartController handler should render with correct context', () => {
+      const h = { view: jest.fn() }
+
+      beforeYouStartController.handler({}, h)
+
+      expect(h.view).toHaveBeenCalledWith(
+        BEFORE_YOU_START_SITE_DETAILS_VIEW_ROUTE,
+        {
+          pageTitle: 'Site details',
+          heading: 'Site details',
+          projectName: 'Test Project'
+        }
+      )
+    })
+
+    test('Should provide expected response and correctly display project name', async () => {
+      const { result, statusCode } = await server.inject({
+        method: 'GET',
+        url: routes.SITE_DETAILS
+      })
+
+      expect(result).toEqual(
+        expect.stringContaining(`Site details | ${config.get('serviceName')}`)
+      )
+
+      const { document } = new JSDOM(result).window
+
+      expect(document.querySelector('h1').textContent.trim()).toBe(
```

Comment:
The [integration test](https://github.com/DEFRA/marine-licensing-frontend/pull/105/files#diff-5c79ed68439dd7e4d885e9c92ed98ff9d7022a39ce18ba1b9caefbe611acd20f) covers this well, maybe at this level it just needs to confirm there's a success response?